src/Appwrite/Auth/Auth.php

```diff
@@ -316,6 +316,10 @@ public static function getRoles(Document $user): array
         }
 
         foreach ($user->getAttribute('memberships', []) as $node) {
+            if (!isset($node['confirm']) || !$node['confirm']) {
+                continue;
+            }
```

Comment:
Is there a way to write test case for this? Would it be possible to create an unconfirmed invitation in the console project and make sure the user does not have access to the team (to see the project)?

Reply:
Good point 👍